gt/main.c

```diff
@@ -1770,3 +1839,46 @@ run_gt(struct net_config *net_conf, struct gt_config *gt_conf)
 	rte_atomic32_init(&gt_conf->ref_cnt);
 	return 0;
 }
+
+void
+update_lua_states(struct gt_config *gt_conf)
```

Comment:
Let's make `update_lua_states()` a Lua C function, so the script that calls it can get the errors that may come up.